scripts/projectUpdater.ps1

```diff
@@ -279,16 +282,17 @@ Write-Host -ForegroundColor DarkGreen "✅ tasks.json"
 
 # TCB does not have the common application Docker files
 if ($templateName -ne "tcb") {
+    # DOCKERFILE.DEBUG:
+    # The generic project doesn't have a Dockerfile.debug
```

Comment:
why not? I have some ideas, like have a debug image that the user can apt-install stuff, change, and we have some command to get this changes and add it to the production image. What do you think?

Reply:
so, this will not be added as multi-container compatible, right?